.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: rust
 sudo: false
+cache: cargo
```

Comment:
The default cargo cache also caches the `target` directory which can unfortunately get quite large due to Cargo not cleaning things out. I wonder if the cache could only be set for the one documentation builder though?

Reply:
I will try this!